docs/index.html

```diff
@@ -4429,7 +4433,7 @@ <h3 class="heading settled" data-level="5.11" id="including"><span class="secno"
 </pre>
    <p>With the above code, you can use <code>[FOO]</code> and <code>[BAZ]</code> macros inside the include file,
 and they’ll be substituted with "bar" and "qux qux qux", respectively.
-(Remember that you can mark text macros as optional by appending a <code>?</code>, like <code>[FOO?]</code>,
+(Remember that you can mark text macros as optional by appending a <code>?</code>, like <code></code>,
```

Comment:
Same problem here, with (ostensible) macros now being replaced inside literals — though, interestingly, `[FOO]` and `[BAZ]` above still weren't replaced.

Reply:
Even more confounding, there's **nothing** different about those two uses. The source for this paragraph, in `index.bs`, is:

```markdown
With the above code, you can use `[FOO]` and `[BAZ]` macros inside the include file,
and they'll be substituted with "bar" and "qux qux qux", respectively.
(Remember that you can mark text macros as optional by appending a `?`, like `[FOO?]`,
in which case they'll be replaced with the empty string if Bikeshed can't find a definition.)
```

Reply:
I guess the difference is that `FOO` and `BAZ` are undefined macros, whereas all of the ones that were replaced were defined. (Even if, as in the case of `FOO?`, they were defined to be the empty string.)